source/features/notifications-filter.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+import {CheckIcon, CheckCircleIcon, XCircleIcon, IssueOpenedIcon, GitPullRequestIcon, GitPullRequestDraftIcon, GitMergeIcon, DotIcon, DotFillIcon} from '@primer/octicons-react';
+
+import features from '.';
+
+const filters = {
+	PR: '[href*="/pull/"]',
+	Issue: '[href*="/issues/"]',
```

Comment:
Not sure if those selectors are robust enough. Is it possible for `/pull/` or `/issues/` to appear in a URL that isn't an issue/PR?

Reply:
Yes it’s possible: 
GitHub.com/fregante/issues (isRepoHome)
GitHub.com/fregante/issues/pulls (isPRList)

Reply:
Both these URLs lead to a 404 page for me (tried with my username too) :thinking:

Reply:
heheh they were examples. I created the repo now so you can see

Reply:
Dammit, thought I could get away with a terse selector :disappointed:\
Guess I have to use a long and ugly `:is()` list now